acceptance/bundle/deployment/bind/job/generate-and-bind/test.toml

```diff
@@ -0,0 +1,8 @@
+Local = false
```

Comment:
Any blockers for Local run?

Reply:
Yes, this test is using a workspace import API to load a notebook file. this API has a complex logic on how to accept notebook files and distinguishes them from regular python files. To succeed locally we would need to replicate this logic in the fake_workspace, which I would argue is out of scope of this PR (the original test is only running in the cloud)

Reply:
It's fine to not do this here. Can you add this context as a comment for someone who would want to make it run locally?